.github/workflows/phpstan.yaml

```diff
@@ -15,5 +15,7 @@ jobs:
                 with:
                     php-version: 7.3
                     coverage: none
-            -   run: composer install --no-progress
-            -   run: composer phpstan
+            -   run: |
+                    composer install --no-progress
+                    composer require staabm/annotate-pull-request-from-checkstyle:^1.0
+            -   run: composer phpstan -- --error-format=checkstyle | vendor/bin/cs2pr
```

Comment:
What is `|` needed for?

Reply:
Some commend in code would help. Mix of phpstan and some style is confusing

Reply:
its a regular unix command pipe. the output of phpstan is piped into the following command (it is passed in as stdin to the cs2pr)

Reply:
Might be better to add this tool to require-dev and and have own composer.jsom script for it.

To prevent hidden install issue and keep consistency to other jobs

Reply:
the tool itself is only usefull in github action context.
therefore I prefer installing it here.

Reply:
I'm against introducing new format for no added value

Reply:
not sure what you mean by 'format'. in case i would add it to the existing phpstan composer command the command could no longer be invoked from the command line.

I would have to create a separate new composer command, which can only be called within github action context (or at least it doesnt provide value if called on the workstation).

if you prefer a separate composer script, I can add it though.

Reply:
The next release of shivammathur/setup-php@v1 will have cs2pr bundled, so we will no longer need to install it via composer require